.github/workflows/test.yml

```diff
@@ -28,112 +28,43 @@ jobs:
           - python-version: 2.7
             django-version: 32
 
-          - python-version: 3.4
-            django-version: 22
-
-          - python-version: 3.4
-            django-version: 30
-
-          - python-version: 3.4
-            django-version: 31
-
-          - python-version: 3.4
-            django-version: 32
+          - python-version: 2.7
```

Comment:
If we've removed Python 2.7 from the list above, why do we need these as 'exclude' rules?